### ISSUE TITLE
[ui5-split-button]: instructions are announced before accessible name

### ISSUE THREAD

**Issue by ndeshev** [UI5/webcomponents#13904]
### Bug Description

Per WCAG 2.5.3 (Label in Name), the accessible name should be announced first so users can immediately identify the control, followed by any additional instructions.

Observed announcement sequence (per button):

`Instructions -> Accessible Name
`

Expected announcement sequence:

`Accessible Name -> Instructions
`

Expected Behavior
For each toolbar button, the screen reader announces the accessible name first, then any descriptive/instructional text, in line with WCAG 2.5.3 (Label in Name).

The ordering defect is inside SplitButton's template, where the computed name is assembled from two hidden spans referenced by aria-labelledby:

```
{/* keyboardHint (instructions) is placed BEFORE accessibleName */}
<span id={`${this._id}-invisibleText`} class="ui5-hidden-text">
  {this.accInfo.keyboardHint} {this.accessibleName}
</span>
```

**Comment by ndeshev** [comment]
1. Hello @UI5/ui5-team-ctr-bal, could you please take a look at this issue, we use the `ui5-split-button` in our RTE component and we received a report about it. Thanks.

**Comment by ndeshev** [comment]
I'm closing this issue as I saw that it is already a duplicated one - https://github.com/UI5/webcomponents/issues/13841
